Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -82,6 +82,7 @@ def tests
         "--format", "ParallelTests::RSpec::RuntimeLogger",
         "--out", "tmp/parallel_runtime_rspec.log"
       ]
+      spec_args << "--tag" << "~needs_official_cmd_taps" if ENV["HOMEBREW_TEST_OFFICIAL_CMD_TAPS"].nil?
```

Comment:
Yeh, I think so. I think this is different to the macOS ones because on Linux you're never going to be hitting that code.